### DISCUSSION TITLE
Struggling to make new pages on grid design

### DISCUSSION THREAD

**Discussion by sagostine** [UNC-Project-Open-AAC/OS-DPI#178] [Help]
Hi Y'all!

I am working on making a interoception grid design but cannot get it to change pages. Either it has only the homepage keys in a grid that I want (in this case body parts) OR it has everything. I think I have the excel sheet right (modeled after the single words with morphs example) so it should change pages but am missing something either in actions or in the layout. Open to suggestions!!

Here is my osdpi file: 
[Introception_Grid.zip](https://github.com/UNC-Project-Open-AAC/OS-DPI/files/11180340/Introception_Grid.zip)

Thanks!
Susie

**Comment by sofia-bg** [comment]
On your layout panel on your grid named hp you have #role equals #role 
you need to change to #role equals $role

**Reply by sofia-bg** [reply]
also in your spreadsheet add hp to the changepg page column in row 69 to make the home pg work on the second level of pgs

**Comment by sagostine** [comment]
Thank you!! It works exactly like I intended. I knew it had to be something simple that I was overlooking!
